src/ripple/rpc/tests/KeyGeneration.test.cpp

```diff
@@ -106,55 +113,82 @@ class WalletPropose_test : public ripple::TestSuite
         expectEquals (result[jss::master_seed_hex], s.master_seed_hex);
         expectEquals (result[jss::public_key], s.public_key);
         expectEquals (result[jss::public_key_hex], s.public_key_hex);
+        expectEquals (result[jss::key_type],
+            params.isMember (jss::key_type) ? params[jss::key_type]
+                                            : "secp256k1");
     }
 
-    void testLegacyPassphrase (char const* value)
+    void testSeed (char const* keyType, key_strings const& strings)
```

Comment:
Is that better?
Should I also change `value` here
https://github.com/wilsonianb/rippled/blob/fix/wallet-propose/src/ripple/rpc/tests/KeyGeneration.test.cpp#L147